essay/swar/latency.md

```diff
@@ -93,13 +98,398 @@ This also shows a notorious deficiency in the efforts to standardize in C++ the
 
 Our SWAR library, in this regard, is "future proof": boolean results are just an specific form of SIMD values.  We use this already for SIMD operations such as saturated addition: unsigned overflow (carry) is detected lane wise and lane-wise propagated to the whole lane, without conversion to scalar.  Saturation is implemented seamlessly as after-processing of computation, without having to convert to "scalar" (the result mask) and back.
 
-In the early planning stages for support of "hardware" SIMD (architectures that do SIMD), I've thought that perhaps we will have to keep our emulated SIMD implementations on "hardware SIMD" as a flavor for users to choose from, since they may be the only way to avoid performance penalties that are higher than the cost of emulation, to prevent the error in the standard library.
+In the early planning stages for support of "hardware" SIMD (architectures that do SIMD), I've thought that perhaps we will have to keep our emulated SIMD implementations on "hardware SIMD" as a flavor for users to choose from, since they may be the only way to avoid performance penalties that are higher than the cost of emulation, to prevent the error in the architecture, an error that no standard library could avoid except synthetizing the operations as we do.
 
 ## A hard challenge
 
 As with much of what superficially looks bad in our SWAR library, upon closer examination, we determine fairly rigorously that these are the best choices, the essential costs.  In that regard, it meets the primary mission of serving as a way to express in software all that we know about performance so that users can benefit the most from our expertise.
 
-[^1] We're beginning to work on explaining the basics of processor instructions and operations costs.
 
 -ed
 
+## Footnotes
+
+[^1] We're beginning to work on explaining the basics of processor instructions and operations costs.
+
+## Appendix, multiplication codegen for x86-64
+
+For this code, multiplying by a compile-time constant, we get this generated code [nice Compiler Explorer link](https://godbolt.org/z/z3b7e1Tnx):
```

Comment:
The example assembly is wildly long: I suspect it would be more illustrative to break out a few critical examples and rely on the godbolt link for those that wish to see them all?